src/containers/SearchPage/SearchContainer.jsx

```diff
@@ -71,7 +71,7 @@ const SearchContainer = ({
     const { levels } = searchParams;
     const subject = data.subjects.find(s => s.id === subjectId);
 
-    const removedFilters = subject.filters?.map(level => level.name) || [];
+    const removedFilters = subject.filters?.map(level => level.id) || [];
```

Comment:
Fikser bug som ikkje nullstillte valgt filter om subject blei fjerna. Dette skyldes at tidligere blei filter håndtert med navn og ikkje id, slik at filter med samme navn i to fag blei sidestillt.